zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -148,6 +154,7 @@ object Server {
   private final case class ConsolidateFlush(enabled: Boolean)                         extends Server[Any, Nothing]
   private final case class AcceptContinue(enabled: Boolean)                           extends UServer
   private final case class FlowControl(enabled: Boolean)                              extends UServer
+  private final case class CustomChannelInitializer(init: Option[ChannelInitializer[Channel]]) extends UServer
```

Comment:
```suggestion
  private final case class UnsafeChannelPipeline(init: ChannelPipeline => Unit) extends UServer
```
This way the user won't need to build the channel pipeline from scratch. They can just re-use the existing pipeline and modify it required.